opcua/common/node.py

```diff
@@ -315,3 +318,9 @@ def call_method(*args, **kwargs):
         from opcua.common import methods
         return methods.call_method(*args, **kwargs)
 
+    def delete_node(self, nodeid, deleterefs=True):
+        deletenode = ua.DeleteNodesItem()
+        deletenode.NodeId = nodeid
+        deletenode.DeleteTargetReferences = deleterefs
+        results = self.server.delete_nodes([deletenode])
+        results[0].check()
```

Comment:
To fit the current API I would rather have a method delete_nodeS([node]). Then it does not really has its place on Node object. he add_xxnodexx methods exists because adding a node should always be done in the context of a parent node . It tried to move all these add_folder/variable methods to functions but it is not really discoverable. Maybe make it a function and  expose these functions on Server objects?
    class Server:
        def create_variable(self, *args):
              create_variable(*args)
        def delete_nodes(self, [nodes])
              delete_nodes([nodes])

            


 Tand because it makes chaining methods easy. But we do not have the same need for delete_nodes

At the node level I would rather see a delete_node(node) method . Not deleting refs here is asking for trouble. the remove delete_child method. USers can just do mys